### PR TITLE
update version to 1.3.0

### PR DESCRIPTION
**Pull request by micahgodbolt**
Just bumping version number to match recent release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 Fabric JS is a JavaScript front-end framework which provides visuals-focused [components](https://github.com/OfficeDev/office-ui-fabric-js/blob/master/ghdocs/COMPONENTSLIST.md) to extend, re-work and use in your Office and Office 365 experiences. Fabric JS uses styles from the [Office UI Fabric Core](https://github.com/OfficeDev/office-ui-fabric-core/) project. Fabric JS is a collection of responsive, mobile-first styles and tools designed to make it quick and simple for you to create web experiences using the Office Design Language. 
 
-**Latest version:** Fabric JS 1.2.0, which uses [Fabric Core](https://github.com/OfficeDev/office-ui-fabric-core) 4.0.0.
+**Latest version:** Fabric JS 1.3.0, which uses [Fabric Core](https://github.com/OfficeDev/office-ui-fabric-core) 5.1.0.
```
Comment (mikewheaton) [inline README.md]: Fabric JS has not yet been updated to use Fabric Core 5.1.0.

Comment (micahgodbolt) [inline README.md]: is bumping JS to 5.1.0 a seperate process?

**Comment by mikewheaton** [comment]
Approved 🐎

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 Fabric JS is a JavaScript front-end framework which provides visuals-focused [components](https://github.com/OfficeDev/office-ui-fabric-js/blob/master/ghdocs/COMPONENTSLIST.md) to extend, re-work and use in your Office and Office 365 experiences. Fabric JS uses styles from the [Office UI Fabric Core](https://github.com/OfficeDev/office-ui-fabric-core/) project. Fabric JS is a collection of responsive, mobile-first styles and tools designed to make it quick and simple for you to create web experiences using the Office Design Language. 
 
-**Latest version:** Fabric JS 1.2.0, which uses [Fabric Core](https://github.com/OfficeDev/office-ui-fabric-core) 4.0.0.
+**Latest version:** Fabric JS 1.3.0, which uses [Fabric Core](https://github.com/OfficeDev/office-ui-fabric-core) 5.0.1.
```
Comment (mikewheaton) [inline README.md]: There shouldn't be any trouble upgrading Fabric JS to use Core 5.1.0 (it's not a breaking release) but it will require another PR.

**Comment by Jahnp** [comment]
Approved

**Pull request merged**